includes/express-checkout/class-wc-payments-express-checkout-button-display-handler.php

```diff
@@ -21,13 +21,6 @@ class WC_Payments_Express_Checkout_Button_Display_Handler {
 	 */
 	private $gateway;
 
-	/**
```

Comment:
See above - removed `WC_Payments_Payment_Request_Button_Handler` as part of the dependencies of `WC_Payments_Express_Checkout_Button_Display_Handler`